core-common/src/main/java/org/zalando/nakadi/service/subscription/zk/ZkSubscriptionClient.java

```diff
@@ -202,21 +203,18 @@ public Partition[] getPartitions() {
         }
 
         public Topology withUpdatedPartitions(final Partition[] partitions) {
```

Comment:
@1u0 could you elaborate on why you change the implementation ?

Reply:
it does not look like related to the change set

Reply:
> * Allows to add new event types to subscription's topology (stored in Zookeeper)
> Note: this is shared code for verified PoC (https://github.com/zalando/nakadi/pull/1554) and the planned rollout approach https://github.com/zalando/nakadi/pull/1558.

It's part of shared change that also needed for #1558.